[ERROR]     Non-resolvable import POM: Could not transfer artifact org.springframework.boot:spring-boot-dependencies:pom:1.3.0.M5 from/to spring-milestones (http://repo.spring.io/milestone): Failed to transfer file: http://repo.spring.io/milestone/org/springframework/boot/spring-boot-dependencies/1.3.0.M5/spring-boot-dependencies-1.3.0.M5.pom. Return code is: 504 , ReasonPhrase:Gateway Time-out. @ line 43, column 25 -> [Help 2]

Ich konnte gerade erfolgreich clean install ausführen...

--- a/vadin-demo-gui/src/main/java/de/muenchen/vaadin/ui/app/views/LoginView.java
+++ b/vadin-demo-gui/src/main/java/de/muenchen/vaadin/ui/app/views/LoginView.java
@@ -94,7 +94,7 @@ public class LoginView extends VerticalLayout implements View {
                 if (security.login(username.getValue(), password.getValue())) {
                     BaseUI.getCurrentEventBus().notify(Key.LOGIN);
                 } else {
-                // Anmeldung fehlgeschlagen
+                    // Anmeldung fehlgeschlagen
                     GenericNotification notif = new GenericWarningNotification("Anmeldung fehlgeschlagen",
                             "Bei der Eingabe Ihrer Usernamens/Ihres Kennworts ist ein Fehler aufgetreten. Versuchen Sie es erneut.");
                     notif.show(Page.getCurrent());

